server/server.go

```diff
@@ -207,6 +207,13 @@ func (s *Server) Run() error {
 				errChan <- err
 				return
 			}
+			err = os.Chmod(s.config.ListenUnix, s.config.ListenUnixMode)
+			if err != nil {
+				s.unixListener.Close();
+				s.unixListener = nil
+				errChan <- err
+				return
+			}
```

Comment:
- Inline the call in the "if"
- no semicolon
- nilling is not necessary, since the program exits

``` go
if err := os.Chmod(s.config.ListenUnix, s.config.ListenUnixMode); err != nil {
  s.unixListener.Close()
  errChan <- err
  return
}
```